upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -976,6 +981,11 @@ func (c *ApplyClusterCmd) validateKubernetesVersion() error {
 	// TODO: make util.ParseKubernetesVersion not return a pointer
 	kubernetesVersion := *parsed
 
+	if c.channel == nil {
+		klog.Warning("unable to load channel. Skipping kubernetes version recommendation/requirements checks", kubernetesVersion)
```

Comment:
Missed this the first time...
```suggestion
		klog.Warning("unable to load channel, skipping kubernetes version recommendation/requirements checks")
```